### PR TITLE
Hyperledger Aries Project Report 2023Q4

### PR DESCRIPTION
**Pull request by swcurran**
FYI @dbluhm @TelegramSam @TimoGlastra @genaris @berendsliedrecht @Patrik-Stas @Moopli @cvarjao

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

**Review by dbluhm — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+[SD-JWTs]: https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/
+
+## Aries Framework Reports
+
+### [Aries VCX]
+
+**Accomplishments**
+
+* Finished migration from legacy indysdk components to maintained components indy-vdr and credx
+* Ongoing effort to add support for V2 versions of issue-credential and presentation protocols to enabler wider variety of VC formats.
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-Aries.md]:
```suggestion
* Ongoing effort to add support for V2 versions of issue-credential and presentation protocols to enable wider variety of VC formats.
```

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+* Big portion of dev effort was dedicated to refactoring, improving developer experience, making tests cleaner, stabler and more efficient.
+* The two ongoing mentorship projects shaping up nicely:
+    * [aries-vcx based mediator service](https://wiki.hyperledger.org/display/INTERN/%5Bdraft%5D+Project+Plan+-+aries-vcx+based+message+mediator+service)
+    * [uniffi based mobile wrapper](https://wiki.hyperledger.org/pages/viewpage.action?pageId=98730187)
+
+**2023Q4 Planning**
+
+* Completion and stabilization of V2 issuance/presentation protocols
+* Stabilization of new reusable components created in 2023, such as did, did_document, did_peer, did_exchange and others. On the same note, general improvement of developer onboarding of these components, as they be adopted also outside of Aries context.
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-Aries.md]:
```suggestion
* Stabilization of new reusable components created in 2023, such as did, did_document, did_peer, did_exchange and others. On the same note, general improvement of developer onboarding of these components, as they will be adopted also outside of Aries context.
```

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  store" for modular extensions to ACA-Py.
+
+[Poetry]: https://python-poetry.org/
+[ruff]: https://docs.astral.sh/ruff/
+
+### [Aries Framework Go]
+
+After an active previous quarter, activity slowed this quarter on [Aries
+Framework Go] for reasons not clear outside of the repository maintainers. We're
+trying get information from the maintainers on their plans for this component.
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-Aries.md]:
```suggestion
trying to get information from the maintainers on their plans for this component.
```

**Review by tkuhrt — APPROVED** [review]
Minor change requests.

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md (hunk 2 above)
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-Aries.md]: Fixed.  Thanks.

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-Aries.md]: Fixed. Thanks.

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md (hunk 3 above)
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-Aries.md]: Updated the status of the Aries Framework Go after connecting with the primary maintainers.

**Comment by swcurran** [comment]
Added an update the Aries Framework Go repository after getting an update from the Maintainers that they would not be continuing to evolve the Framework.

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+None.
+
+# Releases
+
+The following Aries releases occurred in the last quarter:
+
+- [Aries Cloud Agent Python] -- 0.10.0, 0.10.1, 0.10.2, 0.10.3
+- [Aries Framework JavaScript] -- v0.4.0, v0.4.1alpha.5-34, v0.4.1, v0.4.2, v0.5.0alpha57-58
+- [Aries Askar] -- v0.3.0-dev1
+- [Aries VCX Releases] -- v0.58.0-v0.61.0
```
Comment (Patrik-Stas) [inline project-reports/2023/2023-Q4-Hyperledger-Aries.md]: Nitpick: the link doesn't display here, this should be `[Aries VCX]`

Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-Aries.md]: Thanks — fixed.

**Review by denyeart — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+on all of those efforts!
+
+While not a risk to the progress being made in the various open source Aries
+communities, the question of moving some of the Aries projects from the
+[Hyperledger Foundation] to the [Open Wallet Foundation] (OWF) was raised this quarter. A fork
+of a long dormant Aries project (the .NET Framework) has been created in the OWF. A more
+interesting situation is the transfer of the actively maintained [Aries
+Framework JavaScript] repository from Hyperledger to OWF that has been
+[proposed by some Aries Framework JavaScript maintainers]. It's an interesting topic, and
+we'll no doubt see more discussion and action related to Aries and the OWF in coming months.
```
Comment (lehors) [inline project-reports/2023/2023-Q4-Hyperledger-Aries.md:26]: This is definitely a topic the TOC should discuss to develop first an understanding of the dynamic at play and then a position as to where we stand on this issue.

**Review by lehors — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Aries.md (hunk 5 above)
Comment (jimthematrix) [reply project-reports/2023/2023-Q4-Hyperledger-Aries.md:26]: +1

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**